Spatial_Engine/Modify/RoundCoordinates.cs

```diff
@@ -80,11 +79,21 @@ public static IElement2D RoundCoordinates(this IElement2D element2d, int decimal
 
                 return element2d.ISetInternalElements2D(element2d.IInternalElements2D().Select(y => y.ISetOutlineElements1D(y.IOutlineElements1D().Select(x => x.ISetGeometry(Geometry.Modify.IRoundCoordinates(x.IGeometry().IProject(plane), decimalPlaces))).ToList())).ToList());
             }
+            else
```

Comment:
I am not sure if I agree with if/else here: what if there is an edge case of a panel with normal within angle tolerance, but its individual points being snapped in a way that results in a nonplanar surface? One would say this will never happen, then in happens on projects 🙈

To account for the above, can't we simply remove the if part and leave the else, i.e. try rounding and spit error when failed?